README.md

```diff
@@ -107,6 +134,16 @@ This will export the following:
 ```bash
 python import_db.py --profile NEW_DEMO --clusters
 ```
+If you experience errors when you try to import the clusters, it may be that you need to modify the clusters file from the logs directory to include the new instance profile if it's not the same as the one in the old databricks account.
+
+To make changes to a cluster name to match the new databricks account you must edit the clusters log file after export.  You do this by looking at the clusters file and identifying the old cluster instance profile which will include the old account number and the name of the instance profile.
+OLD profile text from an AWS Databricks account:
+arn:aws:iam::111111111111:instance-profile/profileName
```

Comment:
Great suggestion here. I do support an account id change w/ `[--update-account-id UPDATE_ACCOUNT_ID] [--old-account-id OLD_ACCOUNT_ID]` but this does not handle the `profileName` change that you mentioned. It sounds like a common use case for people to change role names. I'll add a new option to support that as well.